e2e-tests/tests.sh

```diff
@@ -20,7 +20,7 @@ finish() {
 }
 
 # ensure that we build docker image in minikube
-[ "$E2E_MINIKUBE_DRIVER" != "none" ] && eval "$(minikube docker-env)"
+[ "$E2E_MINIKUBE_DRIVER" != "none" ] && eval "$(minikube docker-env --shell bash)"
```

Comment:
What is `--shell bash` needed for?

Reply:
It's because `minikube docker-env` by default exports the required commands in the login-shell of the current user. As I use fish-shell, I get fish commands. But this script is executed with `sh` (see the bang) and then it fails. 

With this `--shell bash` I force it to export in a syntax that works with `sh`. I mention this in the commit's long description.